template/razor.go

```diff
@@ -228,6 +232,7 @@ func printDebugInfo(r replacement, content string) {
 	if len(matches) > 0 {
 		logString = logString + " \n"
 	}
-	InternalLog.Trace(logString)
-
+	InternalLog.Debug(logString)
 }
+
+var debugMode bool
```

Comment:
After looking at the uses of debugMode. I'm not sure what it's for. Can you explain?